doc-cheat-sheet.rst

```diff
@@ -0,0 +1,600 @@
+See the `Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#table-directives>`_ for all table syntax alternatives. |x wokeignore:rule=master x|
+
+Both markups result in the following output:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Header 1
+     - Header 2
+   * - Cell 1
+
+       Second paragraph cell 1
+     - Cell 2
+   * - Cell 3
+     - Cell 4
+
+Grid tables
+-----------
+
+See `Grid Tables <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#grid-tables>`_ for reference.
+
+.. code::
+
+   +----------------------+------------+
+   | Header 1             | Header 2   |
+   +======================+============+
+   | Cell 1               | Cell 2     |
+   |                      |            |
+   | 2nd paragraph cell 1 |            |
+   +----------------------+------------+
+   | Cell 3               | Cell 4     |
+   +----------------------+------------+
+
+List tables
+-----------
+
+See `List table <https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table>`_ for reference.
+
+.. code::
+
+   .. list-table::
+      :header-rows: 1
+
+      * - Header 1
+        - Header 2
+      * - Cell 1
+
+          2nd paragraph cell 1
+        - Cell 2
+      * - Cell 3
+        - Cell 4
+
+Notes
+=====
+
+.. list-table::
+   :header-rows: 1
+
+   * - Input
+     - Output
+   * - .. code::
+
+          .. note::
+             A note.
+     - .. note::
+          A note.
+   * - .. code::
+
+          .. tip::
+             A tip.
+     - .. tip::
+          A tip.
+   * - .. code::
+
+          .. important::
+             Important information
+     - .. important::
+          Important information
+   * - .. code::
+
+          .. caution::
+             This might damage your hardware!
+     - .. caution::
+          This might damage your hardware!
+
+Adhere to the following conventions:
+
+- Use notes sparingly.
+- Only use the following note types: ``note``, ``tip``, ``important``, ``caution``
+- Only use a caution if there is a clear hazard of hardware damage or data loss.
+
+Images
+======
+
+.. list-table::
+   :header-rows: 1
+
+   * - Input
+     - Output
+   * - ``.. image:: https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png``
+     - .. image:: https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png
+   * - .. code::
+
+          .. figure:: https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png
+             :width: 100px
+             :alt: Alt text
+
+             Figure caption
+     - .. figure:: https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png
+          :width: 100px
+          :alt: Alt text
+
+          Figure caption
+
+Adhere to the following conventions:
+
+- For local pictures, start the path with :file:`/` (for example, :file:`/images/image.png`).
+- Use ``PNG`` format for screenshots and ``SVG`` format for graphics.
+
```

Comment:
Might also be worth adding a convention on what needs to go in alt text (and what doesn't).

Reply:
Any suggestions? ;)

Reply:
If you want to keep the doc brief and to the point, could just include a link to a place with more info about best practices?

https://abilitynet.org.uk/news-blogs/five-golden-rules-compliant-alt-text might be a decent source